frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -638,17 +638,17 @@ Except when in two columns mode, where this is limited to showing only the previ
         table.insert(menu_items.long_press.sub_item_table, {
             text_func = function()
                 if G_reader_settings:readSetting("highlight_non_touch_interval") == 1 then
-                    return T(_("Interval to speed-up rate: %1 second"), G_reader_settings:readSetting("highlight_non_touch_interval", 1))
+                    return T(_("Interval to speed-up rate: %1 second"), G_reader_settings:readSetting("highlight_non_touch_interval") or 1)
```

Comment:
This and next changes are wrong.

Reply:
How?

Reply:
The or 1 should be inside.

Anyway, make it something like this:
```suggestion
                    local highlight_non_touch_interval = G_reader_settings:readSetting("highlight_non_touch_interval" or 1
                    return T(NC_("Speed-up rate interval: %1 second", highlight_non_touch_interval), highlight_non_touch_interval)
```

Reply:
Wrong
https://github.com/koreader/koreader/blob/ffb8cc53d5abf2c7a51dab12ab60575b7f8b49f1/frontend/apps/reader/modules/readerhighlight.lua#L639-L645
Correct
https://github.com/koreader/koreader/blob/f2691b833e451120f274b285678ef8f7520f31cb/frontend/gettext.lua#L14-L16

Reply:
Shouldn't your NC include a string with seconds in plural ?

Reply:
I did say something like; some assembly required. ;-P

Reply:
there are three opening `(` but only two closing `)`. why did I have to change these anyway? They were fine, it was the ones at 2000-ish that were the problem. these only matter when a user actually comes and edits them.

Reply:
frontend/apps/reader/modules/readerhighlight.lua:641:26: accessing undefined variable 'NC_'